Tests/test_Align_emboss.py

```diff
@@ -226,92 +491,18 @@ def test_pair_example_nobrief(self):
             "|||||:||||||||||||||||| |||||||||||    |||||||||||||:||||||||||||||||||||:|||||||||||||||||||||||  |||||||||||||||:||||||||:|||||||",
         )
 
-    def test_simple_example(self):
```

Comment:
Was the Emboss/alignret.txt test dropped?

Reply:
Yes. Alignret doesn't seem to exist any more in Emboss.